### PR TITLE
Update README.md: preliminary Windows support; executable name change…

### LINKED ISSUE #354: TODO: Update README with simpler instructions for installing dependencies, using script/bootstrap

### PR DESCRIPTION
**Pull request by stephengtuggy**
…s; `vs` and `vsettings` scripts; etc.

Thank you for submitting a pull request and becoming a contributor to the Vega Strike Core Engine.

Please answer the following:

Code Changes:
- [ ] Have the PR Validation Tests been run? See https://github.com/vegastrike/Vega-Strike-Engine-Source/wiki/Pull-Request-Validation
- [X] This is a documentation change only

Issues:
- Please list any related issues
- Fixes #354 
- Partially addresses #123 
- Followup to #53 
- Fixes https://github.com/vegastrike/Assets-Masters/issues/23

Purpose:
- What is this pull request trying to do? Update several sections of the README to reflect the latest information.
- What release is this for? 0.8.x
- Is there a project or milestone we should apply this to? 0.8.x

### CODE REVIEW

**Review by Loki1950 — APPROVED** [review]

**Review by nabaco — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -58,12 +58,12 @@ How to Run
...
 
 - First change settings with `bin/vegasettings`
-- Then run `bin/vegastrike` or double-click on the executable file in the bin directory.
+- Then run `bin/vegastrike-engine` or double-click on the executable file in the bin directory.
 
 Vegastrike command line parameters allow for different start settings and are as follows
 
 ```man
-  vegastrike [-ddata_dir] [-px,y,z] [-jsector/starsystem] missionname
+  vegastrike-engine [-ddata_dir] [-px,y,z] [-jsector/starsystem] missionname
```
Comment (BenjamenMeyer) [inline README.md:66]: `-ddata_dir` is required for the `vegastrike-engine` executable. It's only optional for the `vegastrike` executable.

File: README.md [hunk 2]
```diff
@@ -85,26 +85,33 @@ OPTIONS
...
+bin/vegastrike-engine mission/explore_universe.mission
 ```
 
 is an example of a valid mission call
 
 the -l flag (must be flushed with the system) will force a player to begin in a star system.
 
 ```bash
-bin/vegastrike -lvega_sector/vega mission/bomber.mission
+bin/vegastrike-engine -lvega_sector/vega mission/bomber.mission
```
Comment (BenjamenMeyer) [inline README.md:96]: needs the data_dir option specified in this mode

File: README.md [hunk 3]
```diff
@@ -307,13 +314,19 @@ and run `./script/build.sh -DUSE_SYSTEM_BOOST=NO`
...
 --------------------
 
-Currently VegaStrike is not compiling on Windows. Any help will be appreciated to get it fix. For more information go [here](https://github.com/vegastrike/Vega-Strike-Engine-Source/issues/53)
+Vega Strike is now compiling on Windows! If you want to compile it, try it out, and perhaps offer feedback, that would certainly be welcome. (Installer still pending.)
+
+To compile Vega Strike on Windows, start by installing Visual Studio 2019, if you haven't already. When selecting the Workloads and Components to install, include at least "C++ for Desktop"; "Python"; and a recent build of the Windows SDK. Probably Git and the GitHub for Windows extension also. Install the latest Visual Studio updates as well.
+
+Once Visual Studio finishes installing, and you have rebooted your computer, open a PowerShell window and run `script/bootstrap.ps1`. Once that finishes, run `script/build.ps1`.
+
+Assuming all the above steps succeed, you are now ready to run Vega Strike. Note that `vegasettings` is not currently building on Windows, so you will need to edit `vegastrike.config` manually as needed.
```
Comment (BenjamenMeyer) [inline README.md:323]: I would  add a note that we do not yet have an installer available for Windows.

**Review by BenjamenMeyer — CHANGES_REQUESTED** [review]
Generally fine; though there are a couple minor tweaks needed. Glad to see this happening.

File: README.md (hunk 1 above)
Comment (stephengtuggy) [reply README.md:66]: Good catch

File: README.md (hunk 2 above)
Comment (stephengtuggy) [reply README.md:96]: Another good catch. Thanks

File: README.md (hunk 3 above)
Comment (stephengtuggy) [reply README.md:323]: @BenjamenMeyer There is a note about that further up. I can add one here too if needed.

File: README.md (hunk 1 above)
Comment (stephengtuggy) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (stephengtuggy) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (stephengtuggy) [reply README.md]: Moved note about Windows installer down a paragraph or two. Hopefully it's more noticeable now.

**Review by BenjamenMeyer — APPROVED** [review]

**Comment by stephengtuggy** [comment]
Let's not forget to port this to master.

**Pull request merged**